src/sonic-py-common/sonic_py_common/multi_asic_device_info.py

```diff
@@ -0,0 +1,292 @@
+import glob
```

Comment:
`multi_asic_device_info.py` is a bit long for a module name. Can we shorten it to `multi_asic.py`, or do you have plans to add other multi-ASIC-related modules in the future with different focuses?

Reply:
renamed `multi_asic_device_info.py `to `multi_asic`